robocop/run.py

```diff
@@ -194,10 +194,10 @@ def list_checkers(self):
                     print(f"{rule_def}\n"
                           f"    {configurables}")
                     severity_counter[rule_def.severity.value] += 1
-        suffix = '.' if self.config.list else ' with configurable parameters.'
-        print(f"Altogether {sum(severity_counter.values())} rule(s): "
-              f"{severity_counter['E']} error(s), {severity_counter['W']} warning(s) and {severity_counter['I']} info"
-              f"{suffix}")
+        print(f"Altogether {sum(severity_counter.values())} rule(s) with following severity:\n"
```

Comment:
Have you checked if there is one line of space between this summary and the list of rules?

Reply:
Local version example:
![image](https://user-images.githubusercontent.com/8532066/130066922-8da1d871-5ce8-4dde-88c5-680ba80e9770.png)

Reply:
I mean if there is a blank line between this summary and the actual list of rules while calling `--list`?

Reply:
Ah no, summary is directly under list of rules. If I were to add black line it would be like this:
![image](https://user-images.githubusercontent.com/8532066/130079155-58a914c9-62fb-4438-82b8-bc9a7b4d0149.png)
And now it is:
![image](https://user-images.githubusercontent.com/8532066/130079225-32a9a940-7196-47f5-a321-cd5cc937a182.png)

Which version is better?

Reply:
I think the first one with one blank line of space for readability.